src/web/app/pages-instructor/instructor-session-individual-extension-page/instructor-session-individual-extension-page.spec.ts

```diff
@@ -0,0 +1,432 @@
+    };
+    const testStudent3: Student = {
+      email: 'alex@tmms.com',
+      courseId: 'testId',
+      name: 'alex',
+      teamName: 'Team 1',
+      sectionName: 'Section 1',
+    };
+    const students: Students = {
+      students: [testStudent1, testStudent2, testStudent3],
+    };
+
+    const testInstructor1: Instructor = {
+      name: 'tester1',
+      email: 'tester1@tester.com',
+      googleId: 'instructor-google-id',
+      courseId: 'test-exa.demo',
+      institute: 'institute',
+      joinState: JoinState.JOINED,
+    };
+
+    const testInstructor2: Instructor = {
+      name: 'tester2HasExtension',
+      email: 'tester2@tester.com',
+      googleId: 'instructor-google-id',
+      courseId: 'test-exa.demo',
+      institute: 'institute',
+      joinState: JoinState.JOINED,
+    };
+
+    const instructors: Instructors = {
+      instructors: [testInstructor1, testInstructor2],
+    };
+
+    const testTimeString = '5 Apr 2000 2:00:00';
+
+    let component: InstructorSessionIndividualExtensionPageComponent;
+    let fixture: ComponentFixture<InstructorSessionIndividualExtensionPageComponent>;
+    let studentService: StudentService;
+    let instructorService: InstructorService;
+    let courseService: CourseService;
+    let feedbackSessionsService: FeedbackSessionsService;
+    let timezoneService: TimezoneService;
+    let statusMessageService: StatusMessageService;
+
+    beforeEach(waitForAsync(() => {
+        TestBed.configureTestingModule({
+          imports: [
+            HttpClientTestingModule,
+            RouterModule,
+            InstructorSessionIndividualExtensionPageModule,
+          ],
+          providers: [StudentService, CourseService, TimezoneService, FeedbackSessionsService,
+            {
+              provide: ActivatedRoute,
+              useValue: {
+                queryParams: of({
+                  courseid: testCourse.courseId,
+                  feedbackSessionName: testFeedbackSession.feedbackSessionName,
+                }),
+              },
+            },
+          ],
+        })
+        .compileComponents();
+      }));
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(InstructorSessionIndividualExtensionPageComponent);
+        component = fixture.componentInstance;
+        studentService = TestBed.inject(StudentService);
+        instructorService = TestBed.inject(InstructorService);
+        courseService = TestBed.inject(CourseService);
+        feedbackSessionsService = TestBed.inject(FeedbackSessionsService);
+        timezoneService = TestBed.inject(TimezoneService);
+        statusMessageService = TestBed.inject(StatusMessageService);
+        jest.spyOn(timezoneService, 'guessTimezone').mockReturnValue('UTC');
+        fixture.detectChanges();
+      });
+
+    it('should create', () => {
+        expect(component).toBeTruthy();
+      });
+
+    it('should snap with default fields', () => {
+        expect(component).toBeTruthy();
+    });
```

Comment:
Test does not match description, this should be a snapshot test with default values.